src/text-editor.coffee

```diff
@@ -609,15 +609,19 @@ class TextEditor extends Model
   # * "<filename>" when it is the only editing buffer with this file name.
   # * "<filename> — <unique-dir-prefix>" when other buffers have this file name.
   #
+  # * `textEditors` An optional {Array} of the other {TextEditor} instances that
```

Comment:
I also agree that separating `TextEditor` and `TextEditorPaneItem` makes sense. On the other hand, this is a *huge* change to the API. Just a cursory glance at `Workspace` reveals a bunch of APIs that will need to be touched, altered or rethought.

> Do we have much precedent for this sort of large change?

Only before v1.0. Which makes me feel like we should hold off on splitting these two roles apart until the run up to v2.0.